sample-wasm-chaincode/chaincode_example02/rust/src/lib.rs

```diff
@@ -47,41 +49,43 @@ pub extern "C" fn init(args: i64) -> i64 {
         return -1;
     }
 
-    // Assumption here is account name should not be more than 24 character
-    let first_account = [0; 24];
-    let first_account_name_length;
 
-    let second_account = [0; 24];
-    let second_account_name_length;
+    let first_param_len = unsafe { __get_parameter_size(0) }; //Account 1 name
+    let second_param_len = unsafe { __get_parameter_size(1) }; //Account 1 balance
+    let third_param_len = unsafe { __get_parameter_size(2) }; // Account 2 name
+    let fourth_param_len = unsafe { __get_parameter_size(3) }; // Account 2 balance
```

Comment:
This covers testing, right?

Reply:
Yeah, this one and line 157 for `__get_state_size` function